internal/internal_workflow_testsuite.go

```diff
@@ -229,6 +235,8 @@ type (
 
 		workflowFunctionExecuting bool
 		bufferedUpdateRequests    map[string][]func()
+
+		currentUpdateId string
```

Comment:
What if there are multiple concurrent updates sent to the workflow? It looks like those would be lost.

Reply:
hmm, good point